product-service/src/handlers/getProductsList.ts

```diff
@@ -0,0 +1,13 @@
+import 'source-map-support/register';
+import { APIGatewayProxyHandler } from "aws-lambda";
+import { getProducts } from "../services/products";
+import { response } from './utils';
+
+export const getProductsList: APIGatewayProxyHandler = async () => {
+    try {
+        const products = await getProducts();
+        return response(200, JSON.stringify(products));
+    } catch (e) {
+        return response(200, JSON.stringify(e));
```

Comment:
I think it is worth to think about returning proper response with message something like `Failed to get products`